src/components/notifications/HeiiOnCall.vue

```diff
@@ -9,19 +9,17 @@
         ></HiddenInput>
     </div>
     <div class="mb-3">
-        <label for="heiioncall-trigger-id" class="form-label">{{ $t("Trigger ID") }}<span
+        <label for="heiioncall-trigger-id" class="form-label">Trigger ID<span
             style="color: red;"
         ><sup>*</sup></span></label>
         <HiddenInput
             id="heiioncall-trigger-id" v-model="$parent.notification.heiiOnCallTriggerId" required="true"
             autocomplete="false"
         ></HiddenInput>
     </div>
-    <div class="form-text">
-        <i18n-t tag="p" keypath="wayToGetHeiiOnCallDetails" style="margin-top: 8px;">
-            <a href="https://heiioncall.com/docs" target="_blank">Heii On-Call Documentation</a>
-        </i18n-t>
-    </div>
+    <i18n-t tag="p" keypath="wayToGetHeiiOnCallDetails" class="form-text mt-3">
+        <a href="https://heiioncall.com/docs" target="_blank">{{ $t("documentationOf", ["Heii On-Call"]) }}</a>
```

Comment:
I think adding translations in this way might be easier for our translators

```suggestion
        <template #documentation>
            <a href="https://heiioncall.com/docs" target="_blank">{{ $t("documentationOf", ["Heii On-Call"]) }}</a>
        </template>
```